docker/Dockerfile

```diff
@@ -1,5 +1,13 @@
-FROM ubuntu:22.04
-MAINTAINER Weina Ji <weina.ji@epfl.ch>
+ARG TARGETARCH
+
+FROM ubuntu:22.04 AS linux_amd64
+ARG ARCH=x86_64
+
+FROM ubuntu:22.04 AS linux_arm64
+ARG ARCH=aarch64
+
+FROM linux_$TARGETARCH
+LABEL org.opencontainers.image.authors="Weina Ji <weina.ji@openblueinstitute.org>"
```

Comment:
@WeinaJi I think there's a typo in the email ;)

Reply:
haha, indeed, I will correct that.